routes/addQuestion.js

```diff
@@ -0,0 +1,26 @@
+//jshint esversion:8
+const router = require("express").Router();
+const Action = require("../model/Action");
+const verify = require("./verifyToken");
+
+router.post("/:action_id", verify, async (req, res) => {
+  const ActionID = req.params.action_id;
+  Action.findByIdAndUpdate(
+    ActionID,
+    { $push: { Questions: req.body } },
+    err => {
+      if (err) {
+        res.json(err);
+      }
+    }
+  );
+  await Action.findById(ActionID)
```

Comment:
This func will execute before the previous one completes

Reply:
Working on it